@types/modules/stpm/minievents.d.ts

```diff
@@ -0,0 +1,33 @@
+declare interface METMMiniEventUpdateRequest {
+    eventName?: string;
+    eventDescription?: string;
+    location?: string;
+    virtual?: boolean;
+    joinLink?: string;
+    joinLinkToPresenters?: number;
+    joinLinkToAttendees?: number;
+    companyId?: string;
+    managers?: string[];
+    speakers?: string[];
+}
+
+declare interface METMMiniEventCreateRequest extends METMMiniEventUpdateRequest{
+    eventId: string;
+    eventName: string;
+    eventDescription: string;
+    virtual: boolean;
+}
+
+declare interface METMMiniEvent extends METMMiniEventCreateRequest{
+    location?: string;
+    joinLink?: string;
+    joinLinkToPresenters?: number;
+    joinLinkToAttendees?: number;
+    companyId?: string;
+    managers?: string[];
+    speakers?: string[];
+}
+
+declare interface METMMiniEventList {
+    miniEvents: METMMiniEvent[];

```

Comment:
Heads up! It's the "Mini Events and Workshops Module" (MEWM). Also, I think that these are just referred to as "Events" and not "Mini Events" as they can also include things like deadlines or meals.
```suggestion
declare interface MEWMEventList {
    events: MEWMEvent[];
```